crux_kv/tests/kv_test.rs

```diff
@@ -63,10 +86,12 @@ mod shared {
         }
     }
 
-    #[derive(Effect)]
+    #[derive(Effect, Clone)]
     pub struct Capabilities {
         pub key_value: KeyValue<Event>,
         pub render: Render<Event>,
+        #[effect(skip)]
+        pub compose: crux_core::compose::Compose<Event>,
```

Comment:
Not strictly related to this change, just a thought I had while reading this: it's a bit wasteful to be cloning every single capability when we want to do async things.  Not a big deal, but should we consider making `App::Capabilities` an `Arc<Capabilities>` (at least in our examples, maybe not forcing users to do so)?

Reply:
Yea, this is a fair point, we probably should do something with that. 

Maybe all the examples should also be cloning just the individual capabilities needed in the async block.

Reply:
Yeah, that could also be an approach

Reply:
let me at least make those changes.

Reply:
Right, way less cloning now